app/helpers/admin/editions_helper.rb

```diff
@@ -291,4 +291,23 @@ def specialist_sector_name(sector_content_id)
   def show_similar_slugs_warning?(edition)
     !edition.document.published? && edition.document.similar_slug_exists?
   end
+
+  def edition_is_a_novel?(edition)
+    edition.body.split.size > 99999
+  end
+
+  def edition_has_links?(edition)
+    LinkCheckerApiService.has_links?(edition, convert_admin_links: false)
+  end
+
+  def show_link_check_report?(edition)
+    # There is an edition that is over 200000 words long.
+    # This causes timeouts when LinkCheckerApiService tries to extract links from the body.
+    # This is an exceptional case, but it stops publishers editing their editions.
+    # Short circuit the call to LinkCheckerApiService by testing for an edition being
+    # over 99999 words long (chosen because a novel Wikipedia suggest 100000 words is
```

Comment:
Minor nitpick: should be "chosen because Wikipedia suggests 100000 words is"

Reply:
👍 fixed in https://github.com/alphagov/whitehall/pull/4467/commits/e6050a2c8f4f4de35f3735e729840d693625d988